force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -175,12 +193,13 @@ public with sharing class ClassesObjectsMethods {
      * @return The current date in the current user locale format.
      */
     public static String getCurrentDate() {
-        // Get the todays date
+        // Gets today's date
+        Date todaysDate = date.today();
 
-        // Format the current date
+        // Formats the current date to reflect user locale
+        String adjustedDateFormat = todaysDate.format();
```

Comment:
This is an important method to remember, like String.join, .format() is another oe you will use all the time.